### ISSUE TITLE
Fail to train the spliceai model with tensorflow 2.1.0

### ISSUE THREAD

**Issue by protein2019** [Illumina/SpliceAI#82]
Hi,

I download the training code of SpliceAI from https://basespace.illumina.com/s/5u6ThOblecrh, and train the model following Canonical/README.txt. After training with tensorflow 2.1.0 and keras 2.4.3, the validation set and training set metrics are very low: topkl_accuracy and auprc are around 0.0001. The loss is about 5e-3 after several epochs.

I try with tensorflow 1.4.1 and keras 2.0.5, as described in README.txt, and it works fine. The metrics are quite high and the loss is about 1e-4. Thus, I guess the failure of training may be due to a newer version of tensorflow and keras. 

However, I have to use tensorflow 2, since I am trying to see the influence of some modified layers, which are only accessible in tensorflow 2. So, is there a way to run the training code in tensorflow 2, or perhaps I make some mistakes when training?

Thanks for your help in advance.

**Comment by kishorejaganathan** [comment]
Yes, I have reproduced this issue but am not really sure what the cause is. The solution is adding a batch normalization layer just before the final layer (when using the more recent versions of Keras/Tensorflow).

**Comment by protein2019** [comment]
Thanks a lot. I add a batch normalization layer before the last softmax layer, and it works.
